src/Avalonia.Controls/Flyouts/MenuFlyout.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections;
+using Avalonia.Collections;
+using Avalonia.Controls.Primitives;
+using Avalonia.Controls.Templates;
+using Avalonia.Metadata;
+
+#nullable enable
+
+namespace Avalonia.Controls
+{
+    public class MenuFlyout : FlyoutBase
+    {
+        public MenuFlyout()
+        {
+            _items = new AvaloniaList<object>();
```

Comment:
Is MenuFlyout placement the same as in FlyoutBase be default? I thought it is "Bottom".

Reply:
I think you're right. I forgot to check.

Reply:
Actually, the MenuFlyout page of the XamlControlsGallery shows the MenuFlyout in Top mode, so I guess it does still default to top.